servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultHost.java

```diff
@@ -456,10 +456,13 @@ public int score() {
     @Override
     public String toString() {
         final ConnState connState = this.connState;
-        return "Host{" +
+        String stateString = connState.isUnhealthy() ? State.UNHEALTHY + "(" + connState.healthCheck + ")" :
+                connState.state.toString();
+        return "DefaultHost{" +
                 "lbDescription=" + lbDescription +
                 ", address=" + address +
-                ", state=" + connState.state +
+                ", state=" + stateString +
+                ", healthIndicator=" + healthIndicator +
```

Comment:
Alternatively, we could only report whether it is healthy or not.

Reply:
Let's try what you have now in this PR